src/main/java/frc/robot/subsystems/drivetrain/Chassis.kt

```diff
@@ -362,6 +364,36 @@ object Chassis :
                     FieldCentricFacingAngle.HeadingController.atSetpoint()
             }
 
+    fun driveToChangingPose(
+        pose: () -> Pose2d,
+        vel: () -> Transform2d,
+        endWhenGoal: Boolean,
+    ): Command =
+        primeDriveToPose(pose)
+            .andThen(
+                applyRequest {
+                        val robot = Chassis.state.Pose
+
+                        xController.setGoal(xController.goal.position + (vel().x * 0.02))
+                        yController.setGoal(yController.goal.position + (vel().y * 0.02))
+                        val xOutput = xController.calculate(robot.x) + vel().x
+                        val yOutput = yController.calculate(robot.y) + vel().y
```

Comment:
I think this was to jittery. 

Since we're so close to comp, let's try

```
button.whileTrue(driveToCoralStation.andThen(RobotCentricDrive))
```
That'll give simple, controlled driving while the button is held down. We might want to extend that to the branch alignments as well

Reply:
Oh I see you have that below